docs/user-guide/FAQ.md

```diff
@@ -0,0 +1,37 @@
+# Frequently Asked Questions
+
+## How do I create a pipeline?
+
+To create a pipeline, click the Create icon and paste a Git URL into the form. Followed by `#` and the branch name, if the branch is not `master`.
+
+![Create a pipeline](./assets/create-pipeline.png)
+
+## How do I start a pipeline manually?
+
+To start a build manually, click the Start button on your pipeline page.
+
+![Start a pipeline](./assets/start-pipeline.png)
+
+## How do I update a pipeline repo and branch?
+
+If you want to update your pipeline repo and branch, modify the checkout URL in the Options tab on your pipeline page and click Update.
+
+![Update a pipeline](./assets/update-pipeline.png)
+
+## How do I disable/enable a job temporarily?
+
+To temporarily disable/enable a to quickly stop the line toggle the switch to disable/enable for a particular job under the Options tab on your pipeline page.
```

Comment:
I think you might have a typo in here `To temporarily disable/enable a to quickly stop...`